install/db/dbmigrate_pfile-sha256.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/***********************************************************
+Copyright (C) 2019 Siemens AG
+Author: Sandip Kumar Bhuyan <sandipbhuyan@gmail.com>
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+version 2 as published by the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ ***********************************************************/
+
+use Fossology\Lib\Db\DbManager;
+/**
+ * @file
+ * @brief Migrate DB with new column for pfile table
+ */
+/**
+ * @brief Update the sha256 column of the table with value from textfinding.
+ * @param DbManager $dbManager
+ * @param string $tableName
+ * @return integer Number of entries updated
+ */
+function updateHash($dbManager, $tableName)
+{
+    if($dbManager == null){
+        echo "No connection object passed!\n";
+        return false;
+    }
+    if(DB_TableExists($tableName) != 1) {
+        // Table does not exists (migrating from old version)
+        echo "Table $tableName does not exists, not updating!\n";
+        return 0;
+    }
+    $sql = "SELECT ".$tableName.".".$tableName . "_pk AS id".
+        "FROM $tableName WHERE $tableName.".$tableName."_sha256 is NULL";
+    $statement = __METHOD__ . ".getNullSHA256.$tableName";
+    $rows = $dbManager->getRows($sql, [], $statement);
+    $sql = "UPDATE $tableName
+            SET ".$tableName."_sha256 = $2
+            WHERE " . $tableName . "_pk = $1;";
+
+    $statement = __METHOD__ . ".updateHashOf.$tableName";
+    $dbManager->prepare($statement, $sql);
+    foreach ($rows as $row) {
+        $dbManager->execute($statement, [
+            $row["id"],
+            hash('sha256',  file_get_contents(RepPath($row['id'],"files")))
+        ]);
+    }
+    return count($rows);
+}
+/**
+ * Migration from FOSSology 3.5.0 to 3.6.0
+ * @param DbManager $dbManager
+ * @param boolean $force Set true to force run the script.
+ */
+function Migrate_pfile_sha($dbManager, $force = false)
```

Comment:
Please follow camelcase

Reply:
Do you want me to named it as MigratePfileSHA

Reply:
Since PR #1343 is merged and provides a function to migrate from FOSSology 3.5 to 3.6, will it be possible for you to rebase your branch and use the function `migrate_35_36` in `install/db/dbmigrate_3.5-3.6.php` to hold the logic of migrating the pfile's SHA256?